code/rachel/HTML/lab02.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>G4</title>
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+    <link rel="stylesheet" href="/Users/rcleav/Desktop/PDX_projects/class_swordfish/code/rachel/HTML/lab02_style.css">
```

Comment:
* Need to use a relative path here. It's important to use relative paths so the code works everywhere. See me or Merritt if you want help figuring out how to set it.